demos/common/combined/aws_iot_combined_demo.c

```diff
@@ -67,7 +67,11 @@
 
 /* Amazon FreeRTOS OTA agent includes. */
 #include "aws_ota_agent.h"
-#include "driver/gpio.h"
+
+/* 
+ * Push button HW platform.
+ */
+#include "aws_hw_button.h"
```

Comment:
As discussed, we can just leave this commented out and let users uncomment it. This way, the code will be portable for other platforms besides ESP32